typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -233,12 +231,11 @@ const releaseCandidate: RootAgentConfig = {
       repo,
       tag: '2a16200-20240408-214947',
     },
-    whitelist: releaseCandidateHelloworldMatchingList,
+    // We're temporarily (ab)using the RC relayer as a way to increase
+    // message throughput.
+    // whitelist: releaseCandidateHelloworldMatchingList,
     gasPaymentEnforcement,
-    transactionGasLimit: 750000,
-    // Skipping arbitrum because the gas price estimates are inclusive of L1
-    // fees which leads to wildly off predictions.
-    skipTransactionGasLimitFor: [chainMetadata.arbitrum.name],
+    metricAppContexts,
```

Comment:
nice, so this wouldn't show up in the dashboards because we're filtering by the `hyperlane` context normally, but if we need to we can check RC thanks to this